modules/storages/app/views/storages/admin/storages/show.html.erb

```diff
@@ -50,6 +50,7 @@ See COPYRIGHT and LICENSE files for more details.
   </li>
 <% end %>
 
+<%= content_tag :h3, I18n.t('label_general') %>
```

Comment:
`fieldset`

Reply:
Hi, just FYI, fieldset apparently doesn't work with attributes-key-value in a show.html.erb. Instead, I've used attributes-group--header as in ./app/views/oauth/applications/show.html.erb.